src/components/Signup/Signup.js

```diff
@@ -56,60 +57,61 @@ function Signup() {
   }
 
   return (
-    <div className={styles.container}>
-      <div className={styles.innerBox}>
-        <h1 className={styles.heading}>Sign up</h1>
+    <>
+      <NavBar />
+      <div className={styles.container}>
+        <div className={styles.innerBox}>
+          <h1 className={styles.heading}>Sign up</h1>
 
-        <InputControl
-          label='Email'
-          placeholder='Enter email address'
-          onChange={(event) => setValues((prev) => ({ ...prev, email: event.target.value }))}
-        />
-        <InputControl
-          label='Password'
-          type='password'
-          placeholder='Enter Password'
-          onChange={(event) => setValues((prev) => ({ ...prev, pass: event.target.value }))}
-        />
-        <br></br>
-        {/* TODO: I would suggest we make it select input */}
-        <InputControl
-          label='Gender'
-          placeholder='Enter your gender'
-          onChange={(event) => setValues((prev) => ({ ...prev, gender: event.target.value }))}
-        />
-        {/* TODO: I would suggest we make it calendar input */}
-        <InputControl
-          label='Birth date'
-          placeholder='Enter your birth date'
-          onChange={(event) => setValues((prev) => ({ ...prev, date: event.target.value }))}
-        />
-        {/* TODO: please add a placeholder */}
-        <InputControl
-          label='Height'
-          placeholder='Enter your height '
-          onChange={(event) => setValues((prev) => ({ ...prev, height: event.target.value }))}
-        />
-        {/* TODO: please add a placeholder */}
-        <InputControl
-          label='Weight'
-          placeholder='Enter your weight'
-          onChange={(event) => setValues((prev) => ({ ...prev, weight: event.target.value }))}
-        />
-        <div className={styles.footer}>
-          <b className={styles.error}>{errorMsg}</b>
-          <button onClick={handleSubmission} disabled={submitButtonDisabled}>
-            Sign up
-          </button>
-          <p>
-            Already have an account?{' '}
-            <span>
+          <InputControl
+            label='Email'
+            placeholder='Enter email address'
+            onChange={(event) => setValues((prev) => ({ ...prev, email: event.target.value }))}
+          />
+          <InputControl
+            label='Password'
+            type='password'
+            placeholder='Enter Password'
+            onChange={(event) => setValues((prev) => ({ ...prev, pass: event.target.value }))}
+          />
+          <br></br>
+          {/* TODO: I would suggest we make it select input */}
+          <InputControl
```

Comment:
Just to clarify: are you going to change it to select input in another PR?

Reply:
Will do it later :)